e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -200,4 +200,50 @@ describe("scenarios > embedding-sdk > editable-dashboard", () => {
       cy.findByText("New SQL query").should("not.exist");
     });
   });
+
+  describe("create new question from dashboards", () => {
+    it("should allow creating a new question from the dashboard", () => {
+      cy.get("@dashboardId").then((dashboardId) => {
+        mountSdkContent(<EditableDashboard dashboardId={dashboardId} />);
+      });
+
+      getSdkRoot().within(() => {
+        cy.button("Edit dashboard").should("be.visible").click();
+        cy.button("Add questions").should("be.visible").click();
+        cy.button("New Question").should("be.visible").click();
+
+        cy.log("building the query");
+        H.popover().findByRole("link", { name: "Orders" }).click();
+        /**
+         * We need to visualize before we can save the question.
+         * This will be addressed in EMB-584
+         */
+        cy.button("Visualize").click();
+        cy.button("Save").click();
+
+        H.modal().within(() => {
+          cy.findByRole("heading", { name: "Save new question" }).should(
+            "be.visible",
+          );
+          cy.findByLabelText("Name").clear().type("Orders in a dashboard");
+          cy.button("Save").click();
+        });
+
+        /**
+         * I was supposed to test the dashcard auto-scroll here, but for some reason,
+         * the test always fails on CI, but not locally. So I didn't test it here.
+         */
```

Comment:
This is sad, but I don't see ways to get around this. Here's [the failure](https://github.com/metabase/metabase/actions/runs/16047926309/job/45284326806?pr=60313#step:12:251). It failed completely to scroll on CI. You can download the artifact and see the screenshot.

This worked fine locally for me though 😢